CMSIS/NN/README.md

```diff
@@ -1,18 +1,29 @@
 # CMSIS NN
 
 ## About
-The table below gives  a quick overview of the API's in CMSIS-NN int8 library with symmetric quantization.
+This page  give a quick overview of the functions available and key differences between them.
 
 **Note:** The GitHub documentation does not follow the *develop* branch but rather the last official release in the *master* branch. Consequently, the group documentation linked to in the table table might not have the listed API. Please refer to the description in the [header](https://github.com/ARM-software/CMSIS_5/blob/develop/CMSIS/NN/Include/arm_nnfunctions.h) file instead.
 
+## Legacy vs TFL micro compliant APIs
+There are two kinds of APIs available in the CMSIS-NN repository; One that supports a legacy symmetric quantization scheme[1] and one that supports TFL micro's symmetric quantization scheme. One of the main differences is how the quantization is performed. The legacy APIs have a fixed point format with power of 2 scaling. This simplifies the re-quantization to a cycle efficient shift operation. No new development is done on the legacy functions and all of the new development is on the functions that support TFL micro. The table below highlights some of the differences between the two formats for convolution related functions. The TFL micro compliant APIs in most cases have a _s8 suffix and is always specified in the API header file.
 
+Operation | Legacy APIs | TFL micro compliant APIs|
+|:-----------|:---------------------|:-------------|
+Core loop | No input or filter offset | Input and/or filter offset |
+Re-qunatization | Shift and saturate in one instruction. ~ 5 cycles | Greater than 200 cycles for one output element
```

Comment:
Typo in quantization

Reply:
@christophe0606 Thanks.. updated it now.